app.js

```diff
@@ -12,6 +12,16 @@ app.get('/api', getEndpoints);
 app.get('/api/topics', getTopics);
 app.get('/api/articles/:article_id', getArticles);
 
+app.use((err, req, res, next) => {
+  if (err.code === '22P02') {
+    res.status(400).send({
+      msg: "bad request - invalid SQL",
+      code: err.code
+    });
+  };
```

Comment:
Great work implementing this error handler. Just remove the invalid SQL part and also the error code from this and you are good to go